server/cpho/views/submission.py

```diff
@@ -113,3 +134,25 @@ def get_context_data(self, *args, **kwargs):
         return {
             **super().get_context_data(*args, **kwargs),
         }
+
+    def get(self, request, *args, **kwargs):
+        context = self.get_context_data(**kwargs)
+        if not (
+            test_rule("is_admin", request.user)
+            or (
+                test_rule("is_program", request.user)
+                and test_rule(
+                    "can_submit_as_program", request.user, self.indicator
+                )
+            )
+            or (
+                test_rule("is_hso", request.user)
+                and test_rule(
+                    "can_submit_as_hso", request.user, self.indicator
+                )
+            )
+        ):
+            raise SuspiciousOperation(
```

Comment:
SuspiciousOperation returns a 400 (Bad request) vs PermissionDenied returns 403 (Unauthorized). I don't think anyone in security will care but I think this is more of a 403 than a 400 🤷 

A 400 would be more appropriate if a user bypassed frontend validation like a readonly field or something, we don't use it nearly as often as PermissionDenied